client/windows/settings/SettingsMainContainer.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * SettingsMainContainer.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "widgets/ObjectLists.h"
+#include "widgets/Buttons.h"
+#include "gui/InterfaceObjectConfigurable.h"
+#include "gui/CGuiHandler.h"
+#include "battle/BattleInterface.h"
```

Comment:
Prefer forward declarations in headers
```suggestion
#include "gui/InterfaceObjectConfigurable.h"

class CTabbedInt;
class BattleInterface;
```

Reply:
done